frontend/src/components/authchooser/index.tsx

```diff
@@ -152,7 +152,9 @@ function AuthChooser() {
       haveClusters={!!clusters && Object.keys(clusters).length > 1}
       error={error}
       oauthUrl={`${
-        isDevMode || isElectron() ? 'http://localhost:4466/' : '/'
+        isDevMode || isElectron()
+          ? 'http://localhost:4466' + (getBaseUrl() ? getBaseUrl() : '/')
```

Comment:
I think it should work with two `//` in the URL path. Otherwise let's change it so we don't run the `getBaseUrl()` twice.

Reply:
I changed this around to use a getAppUrl() function and cleaned it up.